src/Compilers/CSharp/Test/WinRT/CodeGen/WinRTCollectionTests.cs

```diff
@@ -253,12 +253,12 @@ public static void Main(string[] args)
 
             verifier.VerifyIL("Class1.Main",
 @"{
-// Code size      213 (0xd5)
-  .maxstack  3
+  // Code size      225 (0xe1)
+  .maxstack  4
```

Comment:
>.maxstack  4 [](start = 2, length = 12)

Why did this change?

Reply:
I would guess its because of the dup that's been added at IL_00ba as part of the conditional access, but someone with more IL knowledge needs to say for sure, I think.

(I'll tidy up the whitespace change - not sure how I missed that)